src/main/java/at/pfeifer/chatapp/controller/ChatController.java

```diff
@@ -74,4 +106,20 @@ public String combineMessageWithSignature(String message, String signature) {
         }
         return String.format("%s | %s", message, signature);
     }
+
+    public String getMessagesFormatted() {
+        return messages
+            .getItems()
+            .stream()
+            .collect(() -> new StringJoiner("\n"), StringJoiner::add, StringJoiner::merge)
+            .toString();
```

Comment:
I think there is a better solution
```suggestion
            .collect(Collectors.joining("\n");
```